src/util/string.cpp

```diff
@@ -467,6 +467,27 @@ u64 read_seed(const char *str)
 	return num;
 }
 
+void parseTextString(const std::string &value, std::string &text, std::string &params, 
```

Comment:
Leave the declaration here and keep the definition in the source file. Headers should be lightweight as possible to for faster compiling.
```C++
void parseTextString(const std::string &value, std::string &text, std::string &params,
		const char sep, const char esc);
```